bin/rector

```diff
@@ -1,14 +1,43 @@
 #!/usr/bin/env php
 <?php
 
-// @see https://github.com/phpstan/phpstan/issues/4541#issuecomment-779434916
-if (
-    file_exists('phar://vendor/phpstan/phpstan/phpstan.phar/stubs/runtime/ReflectionUnionType.php')
-    &&
-    file_exists('phar://vendor/phpstan/phpstan/phpstan.phar/stubs/runtime/Attribute.php')
-) {
-    require_once 'phar://vendor/phpstan/phpstan/phpstan.phar/stubs/runtime/ReflectionUnionType.php';
-    require_once 'phar://vendor/phpstan/phpstan/phpstan.phar/stubs/runtime/Attribute.php';
-}
+/**
+ * @see https://github.com/phpstan/phpstan/issues/4541#issuecomment-779434916
+ *
+ * Point possible vendor locations by use the __DIR__ as start to locate
+ * @see https://github.com/rectorphp/rector/pull/5581 that may not detected in https://getrector.org/ which uses docker to run
+ */
+(static function () {
```

Comment:
This is look like very magically. Class with method would be better.

The `bin/rector` should be clear of any PHP code. It was my mistake to put phar includes here in the first place. Better locatio would be probably the `ContainerFactory`, so any container created is valid without any extra includes.